.changeset/twenty-carrots-jog.md

```diff
@@ -0,0 +1,17 @@
+---
+'@backstage/backend-app-api': patch
+---
+
+Added a configuration to permit backend plugin failures on startup:
+
+```yaml
+backend:
+  ...
+  startup:
+    plugin-x:
+      optional: true
```

Comment:
In a couple of places we have added an extra `plugins:` nesting level under the parent (`startup` in this case). This lets you add defaults and more options under `startup`. Would that make sense here?

Reply:
Yep probably best in this case too, so that we could potentially do things like this?

```
backend:
  startup:
    defaults:
      optional: true
    plugins:
      catalog:
        optional: false
```